data/pathfinder/paizo/campaign_setting/distant_worlds/dw_equip.lst

```diff
@@ -4,5 +4,6 @@
 Pressure Suit			TYPE:Slot_body.Uses_helm_Slot	COST:4000	WT:30						SOURCEPAGE:p.55	QUALITY:Aura|faint abjuration; CL 5th	SPROP:This magical full-body suit completely protects the wearer from temperature extremes (as if by endure elements) and pressure dangers of hard vacuum and all worlds in the system (save the sun), though it only carries enough air for 3 days before it must be recharged in breathable atmosphere (such as that inside a ship).	AUTO:EQUIP|Pressure Suit (Helm)
 Pressure Suit (Helm)		TYPE:Helm.Headgear										SOURCEPAGE:p.55
 
-Rifle.COPY=Shobhad longrifle
-Shobhad longrifle.MOD											RANGE:200	SIZE:L											SPROP:only works in the low gravity of Akiton and similar worlds.
+#Rifle.COPY=Shobhad Longrifle
+#Shobhad Longrifle.MOD											RANGE:200	SIZE:L											SPROP:only works in the low gravity of Akiton and similar worlds.
+Shobhad Longrifle			NAMEISPI:YES					PROFICIENCY:WEAPON|Shobhad Longrifle	TYPE:Weapon.Resizable.Martial.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Rifle		CONTAINS:1|FirearmCartridge=1				COST:5000	WT:12		CRITMULT:x4	CRITRANGE:1	DAMAGE:1d10	EQMOD:Material ~ Steel				WIELD:TwoHandsOnly	RANGE:200	SIZE:L	SOURCEPAGE:p.139	SPROP:Misfire 1; Only functions on low-gravity worlds.
```

Comment:
How is this not working?

Reply:
It works if you load Distant Worlds or Bestiary 4. If you load both you get the error below.
> 21:44:40.208 SEVERE Thread-8 AbstractReferenceManufacturer:927 More than one Equipment with key/name Shobhad Longrifle was built
21:44:40.208 SEVERE Thread-8 AbstractReferenceManufacturer:938 Key case differed for Shobhad longrifle
21:44:40.208 SEVERE Thread-8 AbstractReferenceManufacturer:941 Sources: file:/D:/Apps/RPTools/pcgen/data/pathfinder/paizo/campaign_setting/distant_worlds/dw_equip.lst, file:/D:/Apps/RPTools/pcgen/data/pathfinder/paizo/roleplaying_game/ultimate_combat/uc_equip_arms_armor.lst
21:44:40.264 SEVERE Thread-8 AbstractReferenceManufacturer:458 Reference to Constructed Equipment Shobhad Longrifle is ambiguous
21:44:40.265 SEVERE Thread-8 AbstractReferenceManufacturer:469 Locations: file:/D:/Apps/RPTools/pcgen/data/pathfinder/paizo/campaign_setting/distant_worlds/dw_equip.lst, file:/D:/Apps/RPTools/pcgen/data/pathfinder/paizo/roleplaying_game/ultimate_combat/uc_equip_arms_armor.lst

So, I made them the same. I used the new version that has its own proficiency because the monster entries have weapon focus with the specific type.